src/smt/preprocessor.cpp

```diff
@@ -128,7 +129,8 @@ Node Preprocessor::simplify(const Node& node, bool removeItes)
   Trace("smt") << "SMT simplify(" << node << ")" << endl;
   if (Dump.isOn("benchmark"))
   {
-    Dump("benchmark") << SimplifyCommand(node.toExpr());
+    d_smt.getOutputManager().getPrinter().toStreamCmdSimplify(
+        *d_smt.getOptions().getOut(), node);
```

Comment:
```suggestion
        *d_smt.getOutputManager().getDumpOut(), node);
```

Reply:
Also can make this more compact by caching `OutputManager& om = d_smt.getOutputManager()`.